flink-formats/flink-avro/pom.xml

```diff
@@ -64,16 +64,30 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<!-- use a dedicated Scala version to not depend on it -->
-			<artifactId>flink-table_2.11</artifactId>
+			<artifactId>flink-table-common</artifactId>
 			<version>${project.version}</version>
 			<scope>provided</scope>
-			<!-- Projects depending on this project, won't depend on flink-table. -->
-			<optional>true</optional>
 		</dependency>
 
 		<!-- test dependencies -->
 
+		<dependency>
```

Comment:
what's the test dependency used for?

Reply:
This dependency is necessary for `org.apache.flink.table.descriptors.AvroTest`.